underlay/src/main/resources/config/datamapping/sd/entity/weight/entity.json

```diff
@@ -4,8 +4,8 @@
   "attributes": [
     { "name": "id", "dataType": "INT64", "valueFieldName": "measurement_id" },
     { "name": "person_id", "dataType": "INT64" },
+    { "name": "GRID", "dataType": "STRING", "valueFieldName": "person_source_value"},
     { "name": "is_clean", "dataType": "INT64", "displayFieldName": "is_clean_name", "isComputeDisplayHint": true },
-    { "name": "x_invalid_bool", "dataType": "BOOLEAN" },
```

Comment:
Did you mean to remove this?

Reply:
It possibly was removed from previous sd directory. I checked the ENT_weight in both indexed_sd20230831_beta and indexed_sd20230831 (the previous indexed dataset) and the coldefs match as well as row counts.

Reply:
I just added this added this attribute to height/weight yesterday in PR #734, v0.0.384. It changes the "is measurement clean" attribute from an enum to a boolean, which is more correct based on our conversation at the meeting yesterday. I left the existing enum there as well, so as not to break the UI until the UI switches over to use the boolean. So, it wouldn't have been in the previous index dataset, but I think we should keep it.